src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplateBuilder.java

```diff
@@ -253,7 +262,12 @@ public Pod build() {
         envVars.putAll(jnlp.getEnv().stream().collect(Collectors.toMap(EnvVar::getName, Function.identity())));
         jnlp.setEnv(new ArrayList<>(envVars.values()));
         if (jnlp.getResources() == null) {
-            jnlp.setResources(new ContainerBuilder().editOrNewResources().addToRequests("cpu", new Quantity("100m")).addToRequests("memory", new Quantity("256Mi")).endResources().build().getResources());
+            jnlp.setResources(new ContainerBuilder().editOrNewResources()
+                .addToRequests("cpu", new Quantity(DEFAULT_JNLP_CONTAINER_CPU_REQUEST))
+                .addToLimits("cpu", new Quantity(DEFAULT_JNLP_CONTAINER_CPU_LIMIT))
+                .addToRequests("memory", new Quantity(DEFAULT_JNLP_CONTAINER_MEMORY_REQUEST))
+                .addToLimits("memory", new Quantity(DEFAULT_JNLP_CONTAINER_MEMORY_LIMIT))
```

Comment:
I do not think settings limits is wise. In the case of CPU, this will _prevent_ anything running in the agent container from using more than 10% CPU, arbitrarily slowing down legitimate operations. In the case of memory, this will cause the agent to crash if the JVM happens to use more than 256Mi.

Reply:
Fair enough. I'll remove the limits.